### PR TITLE
Added another example for g:startify_custom_header

### PR DESCRIPTION
**Pull request by karamellpelle**
This example shows a random quote _and_ cow, centered and using a wider speech balloon. Adapted to also work with the newer `cowsay_apj` project (https://github.com/cowsay-org/cowsay).

Also changed temporary `g:script_variable` into `s:script_variable`

Nice plugin, BTW!

### CODE REVIEW

File: doc/startify.txt [hunk 1]
```diff
@@ -637,27 +637,37 @@ Example #2:~
 >
     let g:startify_custom_header =
             \ startify#pad(split(system('fortune | cowsay'), '\n'))
-<
```
Comment (mhinz) [inline doc/startify.txt:640]: Why removing the end marker?

File: doc/startify.txt [hunk 2]
```diff
@@ -637,27 +637,37 @@ Example #2:~
...
 Example #3:~
 
+Random quote with random cow and wider speech balloon, centered (and adapted 
+to fit the current version of the `cowsay-apj` project)
+>
+    let s:header_cmd = 
+      \ 'fortune | cowsay -W 80 -f $(cowsay -l | sed "/[A-Z].*$/d" | shuf -n 1)'
+    let g:startify_custom_header =
+      \ startify#center(split(system(s:header_cmd), '\n'))
+
```
Comment (mhinz) [inline doc/startify.txt:650]: Missing the end marker `<`. It's technically not needed, but conventional.

File: doc/startify.txt [hunk 3]
```diff
@@ -667,7 +677,7 @@ If you provide a string to it instead, Startify will evaluate it every time
 :Startify is run:
 >
     let g:startify_custom_header =
-          \ 'startify#pad(g:ascii + startify#fortune#boxed())'
+          \ 'startify#pad(s:ascii + startify#fortune#boxed())'
```
Comment (mhinz) [inline doc/startify.txt:680]: This will break, because Startify will try to evaluate it outside of the context of the vimrc where `s:ascii` won't exist.

**Comment by mhinz** [comment]
Hi,

I appreciate the PR, but have to decline it.

Apart from the few things I reviewed, I don't think it's an improvement over example `#2`. It merely adds another invocation of `system()` using a cowsay fork. I'm super reluctant in adding more examples when they don't show off something new.

Thanks anyway!

**Comment by karamellpelle** [comment]
Yes, sorry for the pull request, I agree. I started configuring NeoVim in a hypomanic state and began to enjoy lots of plugins like yours. Think less, do more. Got embarrassed later, but decided to just let it go.

**Pull request closed without merging**